src/components/idenfy/IdenfyModal.tsx

```diff
@@ -17,8 +17,8 @@ const IdenfyModal: FunctionComponent = () => {
   );
 
   const handleClose = useCallback(() => {
-    dispatch(setIsIdenfyModalOpen(false));
-  }, [dispatch]);
+    if (isOpen) dispatch(setIsIdenfyModalOpen(false));
```

Comment:
Previously, if you hit the `escape` it would trigger a close of modal bc it is an app level component. I've added this check so that it doesn't trigger it if it's closed.